test/Microsoft.AspNet.Authentication.Test/project.json

```diff
@@ -10,6 +10,7 @@
     "Microsoft.AspNet.Authentication.MicrosoftAccount": "1.0.0-*",
     "Microsoft.AspNet.Authentication.OpenIdConnect": "1.0.0-*",
     "Microsoft.AspNet.Authentication.Twitter": "1.0.0-*",
+    "Microsoft.AspNet.Diagnostics": "1.0.0-*",
```

Comment:
Lame... ok so I guess we just blow up in our tests then?  At least I verified that things work :)